Makefile.PL

```diff
@@ -74,3 +75,37 @@ WriteMakefile(
   dist => { PREOP=>'$(PERL) -MPDL::Core::Dev -e pdlpp_mkgen $(DISTVNAME)' }, # GENERATED subdir in dist tarball
   clean => { FILES => '*~' },
 );
+
+
+sub find_libs {
+  return if $^O !~ /linux/i;
+
+  my @dirs = split /\s+/, `ldconfig -Nv 2>/dev/null | grep : | cut -f1 -d:`;
```

Comment:
So this would only work as the root user, correct? On my system, `ldconfig` is under `/sbin/ldconfig`. If we're running `ldconfig -N`, then probably also use `-X` as well so that the FS isn't touched:

```shell
ldconfig -NXv
```

---

Though I'm not sure this is the best approach. There are ways of getting the library search path from the compiler / linker. Even better is to use `Devel::CheckLib` which is already a configure-time dependency of the dist.

Reply:
I agree with that (and hadn't thought of it). Also, if the `@dirs` stays, it would need to `return` on failure.